app/script/view_model/panel/BasePanelViewModel.js

```diff
@@ -24,20 +24,16 @@ window.z.viewModel = z.viewModel || {};
 window.z.viewModel.panel = z.viewModel.panel || {};
 
 z.viewModel.panel.BasePanelViewModel = class BasePanelViewModel {
-  constructor({isVisible, mainViewModel, navigateTo, onClose, onGoBack, onGoToRoot, repositories}) {
+  constructor({isVisible, navigateTo, onClose, onGoBack, onGoToRoot, repositories}) {
     this.onClose = onClose;
     this.onGoBack = onGoBack;
     this.onGoToRoot = onGoToRoot;
     this.navigateTo = navigateTo;
 
     this.isVisible = isVisible;
 
-    this.repositories = repositories;
-    this.mainViewModel = mainViewModel;
-
-    this.activeConversation = repositories.conversation.active_conversation;
-    this.actionsViewModel = mainViewModel.actions;
-    this.isActivatedAccount = mainViewModel.isActivatedAccount;
+    const conversationRepository = repositories.conversation;
+    this.activeConversation = conversationRepository.active_conversation;
```

Comment:
You could probably inline this without a temp variable.

```suggestion
    this.activeConversation = repositories.conversation.active_conversation;
```